protocols/JabberG/src/jabber_opt.cpp

```diff
@@ -379,7 +378,7 @@ class CDlgOptAccount : public CJabberDlgBase
 		m_btnUnregister(this, IDC_UNREGISTER),
 		m_btnChangePassword(this, IDC_BUTTON_CHANGE_PASSWORD),
 		m_gotservers(false),
-		m_lnkServers(this, IDC_LINK_PUBLIC_SERVER, "https://xmpp.net/directory.php")
+		m_lnkServers("") //

```

Comment:
> You should also remove the line "List of public servers" from https://github.com/miranda-ng/miranda-ng/blob/master/protocols/JabberG/res/jabber.rc, otherwise it is still visible.

This is not enough

OnDropdown
QueryServerListThread
RefreshServersneeds

also should be removed

Reply:
I didn't list all the places, I expect the author to compile it for himself.

Reply:
> You should also remove the line "List of public servers" from https://github.com/miranda-ng/miranda-ng/blob/master/protocols/JabberG/res/jabber.rc, otherwise it is still visible.
> 
> Please make sure everything compiles and looks as expected, before making pull requests, thanks

Yeah, I suppose that this will not so easy and need find more strings and change them.